File: app/src/main/java/com/cashzhang/nozdormu/bean/LoginBody.java
package com.cashzhang.nozdormu.bean;

public class LoginBody {

    private String client_id;
    private String client_secret;
    private String grant_type;
    private String redirect_uri;
    private String code;

    public LoginBody (String code) {
        this.client_id = "feedly";
        this.client_secret = "0XP4XQ07VVMDWBKUHTJM4WUQ";
        this.grant_type = "authorization_code";
        this.redirect_uri = "https://cloud.feedly.com/feedly.html";
        this.code = code;
    }
}

File: app/src/main/java/com/cashzhang/nozdormu/model/LoginBody.kt
package com.cashzhang.nozdormu.model

data class LoginBody(private val code: String) {

    private val client_id = "feedly"
    private val client_secret = "0XP4XQ07VVMDWBKUHTJM4WUQ"
    private val grant_type = "authorization_code"
    private val redirect_uri = "https://cloud.feedly.com/feedly.html"

}